MC/config/QC/json/tpc-qc-standard-direct.json

```diff
@@ -23,7 +23,7 @@
         "url": ""
       },
       "conditionDB": {
-        "url": "ccdb-test.cern.ch:8080"
+        "url": "alice-ccdb.cern.ch:8080"
```

Comment:
Hi @lauraser, sorry, it would not work like that. Could you please remove `:8080` and only keep `alice-ccdb.cern.ch`? The ccdb is listening to the default port, i.e., 80. Thank you.

Reply:
My fault too, I did not spot this mistake, I thought Laura did not add the port.